cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -9,10 +9,21 @@ KUBECONFIG_PATH="${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig"
 
 MASTER_NODE="${CLUSTER_NAME}-control-plane"
 WORKER_NODE_ROOT="${CLUSTER_NAME}-worker"
+NUM_PFS_REQUIRED="${NUM_PFS_REQUIRED:-1}"
 
 OPERATOR_GIT_HASH=8d3c30de8ec5a9a0c9eeb84ea0aa16ba2395cd68  # release-4.4
 SRIOV_OPERATOR_NAMESPACE="sriov-network-operator"
 
+decimal_re='^[0-9]+$'
+if ! [[ "$NUM_PFS_REQUIRED" =~ ${decimal_re} ]]; then
+  echo "FATAL: Wrong value of NUM_PFS_REQUIRED, must be numeric, less or equal to actual PF available"
+  exit 1
+fi
```

Comment:
i prefer to keep it, there is no reason to drop it, i want it to catch erroneous inputs, and that the script will fail as soon as possible.

Reply:
This check is redundant because right after this one there is an if that checks `NUM_PFS_REQUIRED` 
is zero, that if validates for you.
And if the latter check fails this script stops so it wont deploy anything.

Reply:
The later just echo, it doesn't fail.
Check again please.
https://github.com/kubevirt/kubevirtci/pull/510/files#diff-499cff045d8a875e8b39f4a9fd59354eb99e8f8d91a95ebc70fbf42d2d5b5beaR24

Reply:
Sorry I wasn't clear let me explain myself
You can drop the first check, 
and change the second check to `if [ $NUM_PFS_REQUIRED -le 0 ]`
if its true then there is nothing to do.
This way you dont need to validate `NUM_PFS_REQUIRED`.

Reply:
But i think otherwise about blocking zero. as we spoke on other thread, lets keep it there please:
https://github.com/kubevirt/kubevirtci/pull/510#discussion_r549399014

Its a feature imo